implementations/rust/ockam/ockam_api/src/nodes/service/kafka_services.rs

```diff
@@ -292,7 +295,8 @@ impl InMemoryNode {
                 // project identifier
                 self.cli_state
                     .set_policy(
-                        &resources::INLET,
+                        &self.node_name,
+                        &Resource::new(&project),
```

Comment:
Won't it clash with another Inlet we create like this or anything else? Should we pick a more specific Resource name?

Reply:
I think it shouldn't matter in this case, since the policy is hardcoded and not configurable from the client code. 

More specifically, the policy would be shared with the kafka consumers and kafka producers that the current node is running.